modules/charon-core/src/main/java/org/wso2/charon/core/util/AttributeUtil.java

```diff
@@ -110,9 +111,21 @@ public static Date parseDateTime(String dateTimeString) throws CharonException {
      * @param date
      */
     public static String formatDateTime(Date date) {
-        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
-        String formattedDate = sdf.format(date);
-        return formattedDate;
+
+        SimpleDateFormat sdf = new SimpleDateFormat(SCIMConstants.dateTimeFormat2);
+        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
+        return sdf.format(date);
+    }
+
+    /*
+     * SCIM spec requires date time to be adhered to XML Schema Datatypes Specification
+     *
+     * @param date
+     */
+    public static String formatDateTimeLocal(Date date) {
+
```

Comment:
What happens when the date is Null?

Reply:
fixed https://github.com/wso2/charon/pull/270